content/departments/product-engineering/process/planning.md

```diff
@@ -0,0 +1,93 @@
+
+### Good plans are clear and concise
+
+A good plan contains:
+
+- A definition of success that:
+  - Is clear and succinct (so a variety of stakeholders can quickly understand what is important)
+  - Is solution agnostic (so the team has ownership and space to find and pursue the best solution)
+  - Is framed as a customer and/or business problem statement (so we are always working on the right things for our business)
+  - Can be unambiguously measured and evaluated (so there is no disagreement about whether we have achieved success)
+- A concrete set of deliverables we believe will solve the problem (so we can detect/manage dependencies and do capacity planning)
+
+This definition applies recursively and fractically at every layer of a plan (for example: company plan, product/eng department plan, org plan, team plan) for a given unit of time (year, quarter, week).
+
+### Planning is iterative
+
+Arriving at a good plan is an iterative process that is neither exclusively top-down nor exclusively bottom-up because there is a cyclical information dependency. Company level strategy is informed by what is working and not working about our business (context our teams have), and team plans are informed by what our company level strategy is (a decision ultimately owned by the CEO). Fundmentally, everyone on our team has context and insights that are valuable inputs into the planning process.
+
+In practice, both the top-down motion and bottoms-up motion happen concurrently, but we need to bootstrap the process to iteratively work through the information dependency. It is helpful to think about planning as having a `W` shape:
+![Planning shape](https://assets.proof.pub/2056/firstround/O6rbG1rNQ2emg8EPj1Xa_TKImage%200%20Leadership%20Frames.png)
+
+Source: [The Secret to a Great Planning Process — Lessons from Airbnb and Eventbrite](https://review.firstround.com/the-secret-to-a-great-planning-process-lessons-from-airbnb-and-eventbrite)
+
+### Building the delivery plan is part of the work
+
+Once a teams has a clear definition of success, there is usually work that needs to be done to figure out what the delivery plan is to achieve that success. The team might need to collect customer feedback, dig in to metrics, or build prototypes as a first step. This exploration is normal, healthy, and part of the delivery plan!
+
+### Plans are ambitious and feasible
+
+We want our plans to be ambitious so that we challenge ourselves to grow. Plans should be ambitious in a focused direction, not ambitious in the number of things we want to accomplish. Doing one thing exceptionally is better than doing multiple things not exceptionally.
+
+Plans need to be feasible. A plan is feasible only if we can achieve it accounting for realistic conditions and expectations (for example: reserve appropriate amount of time for unplanned work). If a plan requires everything to go perfectly, then it is not feasible (because that is not a realistic expectation).
```

Comment:
Maybe a sentence about how teams would deal with having multiple ambitious goals that are all also at the same time feasible would help connect these more? If we're setting goals fractally at every layer of the organization I'm not totally sure how that would be organized/work. Setting some % of free time for things that we don't know / work that other people are planning for you could be a big unknown.